sdcm/cluster.py

```diff
@@ -1794,6 +1795,7 @@ def install_scylla(self, scylla_repo):
                     apt-get install gnupg1-curl dirmngr -y
                     apt-key adv --fetch-keys https://download.opensuse.org/repositories/home:/scylladb:/scylla-3rdparty-stretch/Debian_9.0/Release.key
                     apt-key adv --keyserver keyserver.ubuntu.com --recv-keys 17723034C56D4B19
```

Comment:
Let’s remove the old key

Reply:
If we remove the old key, upgrade / rollback will fail.

Reply:
Ok, didn’t think about it.
Thanks.